app/src/main/java/com/globant/harrypotterapp/viewmodel/CharactersViewModel.kt

```diff
@@ -35,12 +35,17 @@ class CharactersViewModel(private val getCharactersUseCase: GetCharactersUseCase
             }
         }
     }
+
+    override fun onCharacterClicked() {
+        charactersMutableLiveData.postValue(Event(CharactersData(status = CharactersStatus.OPEN_CHARACTER_DETAIL)))
```

Comment:
you are not into a coroutine, so in this case, you should use charactersMutableLiveData.value =

Reply:
You are right! Refactored!